cfgov/unprocessed/apps/owning-a-home/js/explore-rates/Slider.js

```diff
@@ -0,0 +1,113 @@
+// Required modules.
+import * as atomicHelpers from '../../../../js/modules/util/atomic-helpers';
+import * as standardType from '../../../../js/modules/util/standard-type';
+
+const $ = require( 'jquery' );
+
+import { sliderLabel } from './template-loader';
+import * as domValues from './dom-values';
+import 'rangeslider.js';
+
+/**
+ * Footer
+ * @class
+ *
+ * @classdesc Initializes the organism.
+ *
+ * @param {HTMLNode} element
+ *   The DOM element within which to search for the organism.
+ * @returns {Footer} An instance.
+ */
+function Slider( element ) {
+
+  const BASE_CLASS = 'a-range';
+  const _dom = atomicHelpers.checkDom( element, BASE_CLASS );
+  let _min;
+  let _max;
+  let _onSlideEndHandler;
+  let _sliderTextDom;
+
+  /**
+   * @param {number} min - Minimum value of the range.
+   * @param {number} max - Minimum value of the range.
+   * @param {HTMLNode} sliderTextDom - HTML element used for slider range text.
+   * @param {Function} onSlideEndHandler - Callback when user releases slider.
+   * @returns {Slider|undefined} An instance,
+   *   or undefined if it was already initialized.
+   */
+  function init( min, max, sliderTextDom, onSlideEndHandler ) {
+    if ( !atomicHelpers.setInitFlag( _dom ) ) {
+      return standardType.UNDEFINED;
+    }
+
+    _min = min;
+    _max = max;
+    _onSlideEndHandler = onSlideEndHandler;
+    _sliderTextDom = sliderTextDom;
+
+    return this;
+  }
+
+  /**
+   * @returns {number} Minimum value of the range.
+   */
+  function getMin() {
+    return _min;
+  }
+
+  /**
+   * @returns {number} Maximum value of the range.
+   */
+  function getMax() {
```

Comment:
I don't think using getters / setters makes sense unless you are performing some type of validation or property modification.  It's just more code and it really doesn't protect you. I say, let's convert this to a class and just use the `_` convention.

Reply:
How about this? https://github.com/cfpb/cfgov-refresh/pull/3927/commits/31351f78bb3e96d5c938f6cda06131c27dbe1470